dash/App/Accounts/Add/AddAddress/index.js

```diff
@@ -54,17 +48,30 @@ class AddAddress extends React.Component {
 
   create () {
     this.setState({ index: ++this.state.index })
-    link.rpc('createFromAddress', this.state.address, 'Watch Account', (err) => {
+    const {address: input} = this.state
+    const createAccountCallback =  (address) => link.rpc('createFromAddress', address, 'Watch Account', (err) => {
       if (err) {
         this.setState({ status: err, error: true })
       } else {
         this.setState({ status: 'Successful', error: false })
       }
     })
+
+    if( input.toLowerCase().includes('.eth') ) {
+      link.rpc('resolveEnsName', input, (err, resolvedAddress) => {
```

Comment:
what happens in the UI while the ENS name is resolving? we might want to add some feedback here so the user doesn't think Frame is hanging